Source/Relativity.Transfer.SDK.Samples.Core/appsettings.json

```diff
@@ -58,5 +58,21 @@
     "Source": "",
     // A valid workspace ID (-1 is a default workspace ID).
     "WorkspaceId": -1
+  },
+  "UploadDirectoryBasedOnExistingJob": {
+    // Should be a local directory path.
+    "FirstSource": "",
+    // Should be a local directory path and it should be different than FirstSource.
+    "SecondSource": "",
+    // Should be an UNC format path to the directory on RelativityOne fileshare.
+    "Destination": ""
```

Comment:
Can explain those params? I thought this kind of transfer needs a source (one) and the ID of a download job to get its source as the destination.

Reply:
You are right but @mmkowal-rel do it like this:

- Upload:
    - Upload some data to a destination path using job based workflow (so he need to register this transfer using source and destination) - maybe full path workflow should be used here
    - The second step, which is the main part of the sample, is the transfer of some data set to the same destination path. In that case we use job based workflow.
- Download:
    - Upload some data to a destination path using this time full path workflow
    - The second step, which is the main part of the sample, is download that data set to the local destination path. In that case we use job based workflow